service/history/replicationTaskProcessor.go

```diff
@@ -200,15 +218,52 @@ Loop:
 
 			p.processResponse(response)
 		case <-p.done:
-			p.logger.Info("ReplicationTaskProcessor shutting down.")
 			return
 		}
 	}
 }
 
-func (p *ReplicationTaskProcessor) sendFetchMessageRequest() <-chan *r.ReplicationMessages {
+func (p *ReplicationTaskProcessorImpl) cleanupReplicationTaskLoop() {
+	ticker := time.NewTicker(cleanupInterval)
```

Comment:
Consider adding some gitter so not all the shards will do clean up at the same time?

Reply:
added jitter